packages/admin-ui/ui/public/locales/fr/translation.json

```diff
@@ -0,0 +1,1919 @@
+  "sidebar-store": "Boutique",
+  "sidebar-orders": "Commandes",
+  "sidebar-products": "Produits",
+  "sidebar-categories": "Catégories",
+  "sidebar-customers": "Clients",
+  "sidebar-inventory": "Inventaire",
+  "sidebar-discounts": "Réductions",
+  "sidebar-gift-cards": "Cartes-cadeaux",
+  "sidebar-pricing": "Tarification",
+  "sidebar-settings": "Paramètres",
+  "table-container-soothed-offset_one": "{{soothedOffset}} - {{pageSize}} sur {{count}} {{title}}",
+  "table-container-soothed-offset_other": "{{soothedOffset}} - {{pageSize}} sur {{count}} {{title}}",
+  "table-container-current-page": "{{currentPage}} sur {{soothedPageCount}}",
+  "timeline-request-return": "Demander un retour",
+  "timeline-register-exchange": "Enregistrer un échange",
+  "timeline-register-claim": "Enregistrer une réclamation",
+  "timeline-success": "Succès",
+  "timeline-added-note": "Note ajoutée",
+  "timeline-error": "Erreur",
+  "timeline-timeline": "Chronologie",
+  "upload-modal-new": "nouveau",
+  "upload-modal-updates": "mises à jour",
+  "upload-modal-drop-your-file-here-or": "Déposez votre fichier ici, ou",
+  "upload-modal-click-to-browse": "cliquez pour parcourir.",
+  "upload-modal-only-csv-files-are-supported": "Seuls les fichiers .csv sont pris en charge.",
+  "upload-modal-import-file-title": "Importer {{fileTitle}}",
+  "upload-modal-cancel": "Annuler",
+  "upload-modal-import-list": "Importer la liste",
+  "add-products-modal-add-products": "Ajouter des produits",
+  "add-products-modal-search-by-name-or-description": "Rechercher par nom ou description...",
+  "add-products-modal-cancel": "Annuler",
+  "add-products-modal-save": "Enregistrer",
+  "add-products-modal-product-details": "Détails du produit",
+  "add-products-modal-status": "Statut",
+  "add-products-modal-variants": "Variantes",
+  "templates-general": "Général",
+  "templates-first-name": "Prénom",
+  "templates-last-name": "Nom de famille",
+  "templates-company": "Société",
+  "templates-phone": "Téléphone",
+  "templates-billing-address": "Adresse de facturation",
+  "templates-shipping-address": "Adresse de livraison",
+  "templates-address": "Adresse",
+  "templates-address-1": "Adresse 1",
+  "templates-address-2": "Adresse 2",
+  "templates-postal-code": "Code postal",
+  "templates-city": "Ville",
+  "templates-province": "Province",
+  "templates-country": "Pays",
+  "templates-metadata": "Métadonnées",
+  "collection-modal-success": "Succès",
+  "collection-modal-successfully-updated-collection": "Collection mise à jour avec succès",
+  "collection-modal-error": "Erreur",
+  "collection-modal-successfully-created-collection": "Collection créée avec succès",
+  "collection-modal-edit-collection": "Modifier la collection",
+  "collection-modal-add-collection": "Ajouter une collection",
+  "collection-modal-description": "Pour créer une collection, vous avez seulement besoin d'un titre et d'une poignée.",
```

Comment:
todo: pareil, poignee est une traduction direct, same as the previous comment about it

Reply:
cf. PR description. J'ai essayé de trouver mais je ne vois pas ce qu'on appelle "handle" en anglais. Peut-être "ancre" (anchor) ?